### ISSUE TITLE
S-COM feedback issue.

### ISSUE THREAD

**Issue by Glloyd** [UristMcStation/UristMcStation#219]
Post feedback here.

Signs are needed.

**Comment by AseaHeru** [comment]
Vehicle Depot
Squads
more medical
Robotics
Better research
Shared money in research
Easy way to ship shit from ships to science
Slower mission intervals

**Comment by Glloyd** [comment]
"I lack an account on both help sites listed so my only suggestions are a combat model for the cyborg, a robotics area, make mediborg hyposprays armor piercing, and give the grab command to surgeryborgs somehow."

**Comment by Glloyd** [comment]
"Also the pulse sniper rifle is REALLY broken. It came out liking like a revoler and disappeared after 1 (?) shot."

**Comment by Lance77an** [comment]
Alright. First off, I'd like to thank all you lovely people for making this.

Now.
Squads - Preferably with structure, such as Leader, Medic, Heavy, Sniper, Assault. Perhaps a specific shuttle for each squad or two squads?
Medical Bay - Lots of improvement here. Open layout, more advanced scanners, more sleepers, more cryochambers(Perhaps with a more efficient mix?), more Surgery rooms,  more everything. Perhaps also add something chemistry related?
Vehicles - Cool and all, I enjoy them. Give them a special shuttle, or make only one fit on each squad-shuttle. More organized system for getting them from the factories into the docking areas and to the front lines. 
Friendly Fire - If not hardcoding a decrease to this, at least give us equipment that protects us from the weapons we carry. 
Post-Launch joining - If you wish to get to a mission after launch, I recommend having three or four little 'Pods' that launch you to the site of the current mission. Aka just teleport you to the current location of the squad ship the pod is designated to.
Weapons - Decent as they are. Add attachments, especially a light. Melee weapons also would be nice. Maybe a hardcoded less damage to team-members?
Healing - Streamline it for on-site healing, if possible? I dunno. Maybe just...
Shuttles - MEDICAL BAY ON EACH SHUTTLE, PLEASE. That would be amazing. 
Medics - From my experience, our field medics are far under equipped. A couple cyrobags, some pills, more medical supplies, a backpack to hold said supplies, and a damn medical HUD would be very nice. At least make said things re searchable.

Thats my first round of suggestions. Love you all, had TONS of fun, and hope to see this come to life. Thanks for your hard work, guys.

**Comment by AseaHeru** [comment]
Actually, with a military belt(Which can hold medkits, guns, whatever) and a medical ERT suit, one medic can do nearly anything. Well, except surgery...

**Comment by scrdest** [comment]
Most of the jumpsuit-slot clothes were broken. Even as minor as colored jumpsuits. under/rank/ clothing remains mostly unaffected - working ones include Chaplain, Detective's default Hard-Worn Suit, HoP's and a bunch of other ones, but I think even there some are broken. The icons are OK, but the on-mob sprites are unavailable.

Tested in S-Com, compiled and ran in 507, gamemode force quickstarted.

**Comment by scrdest** [comment]
Even weirder issue - changing the gamemode to Extended then starting it announces it as S-COM round.

**Comment by scrdest** [comment]
OK, THIS IS SOME TOP-PRIORITY SHIT - IT'S NOT EVEN JUST CLOTHES. ITEM SPRITES MAY BE BROKEN AS WELL, REVERT ASAP!

**Comment by scrdest** [comment]
KILL ME PLEASE. THE FUCKING GRENADE RUNTIMES ARE GONE ON 506 NOW. SO IS THE ICONS ISSUE.

**Comment by Glloyd** [comment]
BYOOOOOOOOND

**Comment by Glloyd** [comment]
What did you do to make them gone?

**Comment by scrdest** [comment]
A blood sacrifice to the Geometer of Code.

Joking aside, nothing at all. It just disappeared after I compiled the code at the most recent Urist version, so after all your updates since I was testing it.

**Comment by TGameCo** [comment]
Byond originated when a magnet passed over a floppy disk, or so the legend, and this scenario that supports this, goes

**Comment by Glloyd** [comment]
OKAY

@scrdest, I'm in the middle of exams, so I won't have a chance, but can you try it on 507.1278 and tell me how it goes? Also, no grenade runtimes are because I moved the cleaner grenades to boxes. Place one on the map, compile and then go from there. I want to get this shit working again.

**Comment by Glloyd** [comment]
Second test is tonight (Sunday April 19th) at 6PM EST

**Comment by scrdest** [comment]
Enemy mobs need a major slowdown, at least the ranged ones, sanic melee drones may add to the panic.

3fast enemies are the main cause for the friendly fire, as people reflex-fire and reposition all the time, at the same time feeling rather spread out - the missions seem to end very quickly.

**Comment by Glloyd** [comment]
Missions being quick was partially me speeding things through, but enemies definitely need a slowdown.

**Comment by TGameCo** [comment]
Yeah, the last time I played the enemies ran incredibly fast, and you really couldn't shoot them unless you got them onto another person, or just sat there and tried to tank the damage

**Comment by AseaHeru** [comment]
Squad captains have minimal access, less than squaddies.
Borgs/robots/androids spawn without a powercell.

**Comment by Glloyd** [comment]
People with glasses disability need to spawn with glasses, Unathi need sandals

**Comment by Glloyd** [comment]
Medics need better shit for bone breaks.

**Comment by scrdest** [comment]
You know, re: mobs' speed, changing it for mobs is trivial, but it might be interesting to give basic S-COM gear a slowdown effect to make the ranged combat more... well, ranged, so that both sides' basic assault troopers are blasting at each other with their Spess-Rifles, while making Snipers a regular round or just relatively boosted speed. 

That would fix the yakketyness and differentiate classes a bit more, making Assault the jack of all trades, Snipers into a, to use TvTropes naming, Glass Cannon, CQC into a Mighty Glacier, and Medics into a specialist.

**Comment by Glloyd** [comment]
That was the attempt for classes. The classes more or less fit into those roles, and things kind of even out in the third tier. Sniper is weak armour wise along with medics, and Heavy/Assault get more damaging guns and better armour, but deal with more slowdown. Heavy armour in particular has a crippling slowdown, but it's damn good against melee. Re: Ranged combat, if you can make the test on Friday, there'll be a lot of changes to the AI and they should hang back and shoot instead of bumrushing everyone.

**Comment by scrdest** [comment]
Yeah, I know, I just meant to say to say is to slash it across the board so neither side is sanic by default and spread the classes out on the now-broader speed chart between max and lowest speeds. If that's a thing, well, I believe the relevant answer is 'durr hurr'.

**Comment by scrdest** [comment]
Make radios work planetside. Possibly with squad channels.

**Comment by Glloyd** [comment]
Thank you scrdest.You're my hero.

On Sat, May 9, 2015 at 6:17 AM, scrdest <notifications@github.com> wrote:

> Make radios work planetside. Possibly with squad channels.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/219#issuecomment-100459690>
> .
>



-- 
Graham Lloyd

**Comment by scrdest** [comment]
Now that I think of it - it would be nice if, for the open maps, there was some kind of a token border - say, a line of shrubbery alongside the forest/jungle border or some kind of a wire fence around the city.

**Comment by Glloyd** [comment]
If I had sprites for that, I'd do that.

On Sat, May 16, 2015 at 5:38 PM, scrdest <notifications@github.com> wrote:

> Now that I think of it - it would be nice if, for the open maps, there was
> some kind of a token border - say, a line of shrubbery alongside the
> forest/jungle border or some kind of a wire fence around the city.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/219#issuecomment-102700263>
> .
>



-- 
Graham Lloyd

**Comment by scrdest** [comment]
Suggestions from the round:
* Anti-tank aliums
* Two missions simultaneously.

**Comment by Glloyd** [comment]
it was an issue in hostile.dm, i've fixed it now, aliens should attack tansk

**Comment by scrdest** [comment]
I intended it more as a specialized anti-tank, a unit that renders CVRs less of a 'shoot all the aliums until it explodes' thing, like an RPG trooper.

**Comment by Glloyd** [comment]
Ahhh. I could probably do something like that.

**Comment by TGameCo** [comment]
If you insert
```
client
    perspective = EDGE_PERSPECTIVE
```
somewhere in the code, It'll prevent the camera from showing that annoying black void after the map ends, and just keep the camera inside the map

**Comment by scrdest** [comment]
Maybe we should code in something a'la Scan For Activity from NewCom, except for missions - let the troops have some downtime.

**Comment by Glloyd** [comment]
Already planned. It's one of the three major updates I was talking about.

On Sat, Jun 13, 2015 at 5:36 PM, scrdest <notifications@github.com> wrote:

> Maybe we should code in something a'la Scan For Activity from NewCom,
> except for missions - let the troops have some downtime.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/219#issuecomment-111754174>
> .
>



-- 
Graham Lloyd

**Comment by scrdest** [comment]
Also: Abort button for the Commander, once radios work on and off missions.

**Comment by scrdest** [comment]
People don't seem to do feedback much, so I'm relaying what I heard on the server - a lot of people, including a good portion of staff, complain the mode with current amount of missions is too long.

**Comment by Glloyd** [comment]
Wow. In early tests people complained it was too short. So what should be
changed? The amount of missions? Or the time between missions? It's 8
minutes or so between missions.

On Thu, Jun 25, 2015 at 2:04 PM, scrdest <notifications@github.com> wrote:

> People don't seem to do feedback much, so I'm relaying what I heard on the
> server - a lot of people, including a good portion of staff, complain the
> mode with current amount of missions is too long.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/219#issuecomment-115346488>
> .
>



-- 
Graham Lloyd

**Comment by scrdest** [comment]
Amount of missions, it seems, it's the thing that eats up most time.

**Comment by scrdest** [comment]
Mapping: S-COM base needs some tools. Just the regular ol' welders, crowbars and what-have-you.

**Comment by Glloyd** [comment]
They're there.

On Sat, Jun 27, 2015 at 1:50 PM, scrdest <notifications@github.com> wrote:

> Mapping: S-COM base needs some tools. Just the regular ol' welders,
> crowbars and what-have-you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/219#issuecomment-116108415>
> .
>



-- 
Graham Lloyd

**Comment by scrdest** [comment]
Exosuit fabricators to fix mechs are needed, apparently.
